app/assets/javascripts/discourse/app/components/composer-editor.js

```diff
@@ -720,21 +728,48 @@ export default Component.extend(ComposerUpload, {
       return;
     }
 
-    const parentContainer = $(event.target).closest(
+    const buttonWrapper = $(event.target).closest(".button-wrapper");
+    const imageResize = buttonWrapper.find(".scale-btn-container");
+    const readonlyContainer = buttonWrapper.find(
       ".alt-text-readonly-container"
     );
-    const altText = parentContainer.find(".alt-text");
-    const correspondingInput = parentContainer.find(".alt-text-input");
+    const altText = readonlyContainer.find(".alt-text");
+
+    const editContainer = readonlyContainer.next();
+    const editContainerInput = editContainer.find(".alt-text-input");
 
-    $(event.target).hide();
-    altText.hide();
-    correspondingInput.val(altText.text());
-    correspondingInput.show();
+    imageResize.hide();
+    readonlyContainer.hide();
+    editContainerInput.val(altText.text());
+    editContainer.removeAttr("hidden");
```

Comment:
I can't use `.show()` as jQuery sets `display: inline`, which worked before because the way elements were nested was different.